src/scenes/game-scene.ts

```diff
@@ -46,29 +48,61 @@ const levelConfig = {
   ],
 };
 
+const soundConfig = [
+  {
+    key: 'punch',
+    filePath: 'audio/sfx/punch.wav',
+    loop: false
+  },
+  {
+    key: 'music1',
+    filePath: 'audio/music/Fluffing-a-duck.mp3',
+    loop: true
+  },
+  {
+    key: 'music2',
+    filePath: 'audio/music/Sneaky-snitch.mp3',
+    loop: true
+  },
+  {
+    key: 'music3',
+    filePath: 'audio/music/spook.mp3',
+    loop: true
+  }
+];
+
 class GameScene extends Phaser.Scene {
   private parallax: Parallax | undefined;
 
   private level: Level | undefined;
 
+  private audio: Audio | undefined;
+
   constructor() {
     super('scene-game');
   }
 
   preload() {
     Parallax.preload(this, parallaxName);
     Level.preload(this, levelConfig);
+    Audio.preload(this, soundConfig);
   }
 
   create() {
     // toggle debug GFX
     // toggleDebug(this);
     this.input.keyboard?.on('keydown-CTRL', () => toggleDebug(this));
+    this.input.keyboard?.on('keydown-E', () => {
+      this.audio?.playAudio("punch");
+    });
 
     this.matter.add.mouseSpring();
 
     this.parallax = new Parallax(this, parallaxName);
     this.level = new Level(this, levelConfig);
+    this.audio = new Audio(this, soundConfig);
+
+    this.audio?.playAudio("music2");
```

Comment:
its not linted :thinking: 
please run `npm run lint` and commit the changes (mapData1.json is also not linted, my bad)
also i will help with that setup on your local next couple of days